cli/app/common.go

```diff
@@ -0,0 +1,43 @@
+package app
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+//	"github.com/docker/libcompose/newcli/app"
+	"github.com/docker/libcompose/project"
+)
+
+// Populate updates the specified project context based on command line arguments and subcommands.
+func Populate(context *project.Context, c *cobra.Command) {
```

Comment:
We have to work to remove the need for this too, I still didn't think how (and maybe it's not in this PR's scope to keep the changes small) :stuck_out_tongue_closed_eyes: